stencil.conf.js

```diff
@@ -26,6 +22,9 @@ var watchOptions = {
  * Watch any custom files and trigger a rebuild
  */
 function development() {
+    var webpack = require('webpack');
```

Comment:
@jbruni can we just move the `var webpack = require('webpack');` outside so we don't have to specify it for each env.